src-conf/mssql.sql

```diff
@@ -1327,12 +1327,12 @@ create table workflow_task (
    mod_date datetime null,
    due_date datetime null,
    created_by NVARCHAR(255) null,
-   assigned_to NVARCHAR(255) null,
+   assigned_to NVARCHAR(36) null,
    belongs_to NVARCHAR(255) null,
    title NVARCHAR(255) null,
    description NVARCHAR(MAX) null,
-   status NVARCHAR(255) null,
-   webasset NVARCHAR(255) null,
+   status NVARCHAR(36) null,
+   webasset NVARCHAR(36) null,
```

Comment:
Yeah I know, but the others DB doesn't seem to care about the diff between the length. E.g: 
Foreign-key constraints:
```
    "fk_workflow_assign" FOREIGN KEY (assigned_to) REFERENCES cms_role(id)
    "fk_workflow_step" FOREIGN KEY (status) REFERENCES workflow_step(id)
    "fk_workflow_task_asset" FOREIGN KEY (webasset) REFERENCES identifier(id)
```